website/docs/observability/index.md

```diff
@@ -13,6 +13,15 @@ In this chapter,  we'll cover how you can use AWS observability solutions integr
 * Monitoring Metrics with CloudWatch Container Insights
 * Monitoring EKS Metrics with AMP and ADOT.
 
+:::tip If you did not a complete the [Introduction Getting Started session](../introduction/getting-started/finish.md) you can  
```

Comment:
This is not necessary since it isn't a lab

Reply:
Done